### PR TITLE
add unit tests, remove hardcode STDOUT, add client-id

### PR DESCRIPTION
**Pull request by philbrookes**
**Describe what this PR does and why we need it**:
- Improve the coverage of the get client configs command
- Remove the hardcoded output to STDOUT
- Add --client-id to the client configs command

**Which issue this PR fixes (This will close that issue when PR gets merged)**
fixes AG-1912

### CODE REVIEW

File: pkg/cmd/clientConfig.go [hunk 1]
```diff
@@ -70,18 +78,41 @@ kubectl plugin mobile get clientconfig`,
...
+				return errors.Wrap(err, fmt.Sprintf(output.FailedToOutPutInFormat, "ServiceConfig", outputType(cmd.Flags())))
 			}
-			return err
+			return nil
 		},
 	}
+
+	ccc.Out.AddRenderer("get"+cmd.Name(), "table", func(writer io.Writer, serviceConfigs interface{}) error {
+		scL := serviceConfigs.(ServiceConfigs)
+		var data [][]string
```
Comment (witmicko) [inline pkg/cmd/clientConfig.go]: could you please rename `scL` it is hard to know what it is without looking it up.

File: pkg/cmd/clientConfig.go [hunk 2]
```diff
@@ -16,24 +16,33 @@ package cmd
...
 		k8Client: k8Client,
+		BaseCmd:  &BaseCmd{Out: output.NewRenderer(out)},
 	}
 }
 
+// GetClientConfigCmd returns a cobra command object for getting client configs
 func (ccc *ClientConfigCmd) GetClientConfigCmd() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "clientconfig",
```
Comment (maleck13) [inline pkg/cmd/clientConfig.go]: can we update the the command to use a similar approach as this command https://github.com/aerogear/mobile-cli/blob/master/pkg/cmd/clients.go#L86

File: pkg/cmd/clientConfig.go [hunk 3]
```diff
@@ -70,18 +78,41 @@ kubectl plugin mobile get clientconfig`,
...
+		}
+		table := tablewriter.NewWriter(writer)
+		table.SetAlignment(tablewriter.ALIGN_LEFT)
+		table.AppendBulk(data)
+		table.SetHeader([]string{"Name", "config"})
+		table.Render()
+		return nil
+	})
+
+	cmd.PersistentFlags().StringP("client-id", "c", "", "--client-id=cfd0128d28e9f9d91238dbc")
```
Comment (maleck13) [inline pkg/cmd/clientConfig.go]: myabe change the usage here to just be ```"--client-id=<clientID>"```

**Comment by maleck13** [comment]
Do we want to add a test for when there no services provisioned to ensure it responds in the expected way?
Also I think as the command changed there is a change needed to the readme that shows some example commands

**Comment by philbrookes** [comment]
@maleck13 updated this now, if you can take another look please.

**Pull request merged**